docs/api.md

```diff
@@ -7,26 +7,144 @@ Loosely based off the example in the README
 
 | Name | Type | Description | Example |
 | ------- | ------- | ------- | ------- |
-| **[black_id](#resource-player)** | *number* | this could be a uuid to make it db agnostic | `42.0` |
-| **date_played** | *date-time* |  | `"2015-01-01T12:00:00Z"` |
-| **game_record** | *string* |  | `"<raw sgf string>"` |
-| **game_server** | *string* |  | `"KGS"` |
-| **rated** | *boolean* |  | `true` |
-| **[white_id](#resource-player)** | *number* | this could be a uuid to make it db agnostic | `42.0` |
+| **[black_id](#resource-player)** | *integer* | black player's id | `42` |
+| **date_played** | *date-time* | time the game was played | `"2015-01-01T12:00:00Z"` |
+| **game_record** | *string* | game record in SGF | `"<raw sgf string>"` |
+| **game_server** | *string* | game server id | `"KGS"` |
+| **id** | *integer* | this could be a uuid to make it db agnostic | `42` |
+| **rated** | *boolean* | whether this was a rated game (on the server?) | `true` |
+| **[white_id](#resource-player)** | *integer* | white player's id | `42` |
+
+### Game Creation
+
+Report a game result
+
+```
+POST /api/v1/games
+```
+
+#### Required Parameters
+
+| Name | Type | Description | Example |
+| ------- | ------- | ------- | ------- |
+| **black_id** | *integer* | black player's id | `42` |
+| **date_played** | *date-time* | time the game was played | `"2015-01-01T12:00:00Z"` |
+| **game_record** | *string* | game record in SGF | `"<raw sgf string>"` |
+| **game_server** | *string* | game server id | `"KGS"` |
+| **rated** | *boolean* | whether this was a rated game (on the server?) | `true` |
+| **white_id** | *integer* | white player's id | `42` |
+
+
+
+#### Curl Example
+
+```bash
+$ curl -n -X POST http://dev.usgo.org/api/v1/games \
+  -d '{
+  "black_id": 42,
+  "white_id": 42,
+  "game_server": "KGS",
+  "rated": true,
+  "date_played": "2015-01-01T12:00:00Z",
+  "game_record": "<raw sgf string>"
```

Comment:
I wasn't aware they used a different go record format. That would explain why the few .sgf's I see come out of asia are inevitably badly formatted - if it were used more frequently, stuff like that would have been ironed out.

Let's figure out the format conversion when we get to that point. When we start talking, it could be as simple as asking, "hey, do you know what SGF is?" and them saying "yes"